### PR TITLE
Update UiLadderLevel struct

### PR DESCRIPTION
**Pull request by jarry-xiao**

### CODE REVIEW

File: typescript/phoenix-sdk/src/market.ts [hunk 1]
```diff
@@ -113,9 +113,14 @@ export type L3UiBook = {
   asks: L3UiOrder[];
 };
 
+export type UiLadderLevel = {
+  price: number;
+  quantity: number;
```
Comment (eugene-chen) [inline typescript/phoenix-sdk/src/market.ts:118]: i think we use priceAsFloat and sizeInRawBaseUnits elsewhere

File: typescript/phoenix-sdk/src/utils/market.ts [hunk 2]
```diff
@@ -459,15 +460,17 @@ function levelToUiLevel(
...
   sizeInBaseLots: BN,
   quoteAtomsPerQuoteUnit: number
-): [number, number] {
-  return [
-    ((toNum(priceInTicks) / quoteAtomsPerQuoteUnit) *
-      marketData.quoteLotsPerBaseUnitPerTick *
-      toNum(marketData.header.quoteLotSize)) /
+): UiLadderLevel {
+  return {
+    price:
```
Comment (eugene-chen) [inline typescript/phoenix-sdk/src/utils/market.ts:465]: use conversion method here?

**Review by eugene-chen — APPROVED** [review]

File: typescript/phoenix-sdk/src/market.ts (hunk 1 above)
Comment (romanR1H2) [reply typescript/phoenix-sdk/src/market.ts:120]: Gi

**Review by romanR1H2 — APPROVED** [review]
rust/Cargo.lock

**Pull request merged**